SPECS/libseccomp/libseccomp.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        Enhanced seccomp library
 Name:           libseccomp
-Version:        2.5.4
+Version:        2.5.5
```

Comment:
Was this version bump required to build containerd/runc?

Reply:
Yes, runc requires libseccomp 2.5.5, and [libseccomp v2.5.4](https://github.com/seccomp/libseccomp/releases/tag/v2.5.4) is no longer supported upstream, a more recent release is suggested to use. I will add the version bump of libseccomp and runc into the containerd changelog as well.